com.unity.render-pipelines.universal/Runtime/Passes/PostProcessPass.cs

```diff
@@ -19,7 +19,7 @@ namespace UnityEngine.Rendering.Universal.Internal
     /// <summary>
     /// Renders the post-processing effect stack.
     /// </summary>
-    public class PostProcessPass : ScriptableRenderPass
+    internal class PostProcessPass : ScriptableRenderPass
```

Comment:
you should change the namespace as well to be `UnityEngine.Rendering.Universal`. We should actually get rid of `UnityEngine.Rendering.Universal.Internal` for all passes, but I recommend to be done in another PR.

Reply:
By default classes are internal, so maybe you only want to remove the public